extensions/amp-script/0.1/test/unit/test-user-activation-tracker.js

```diff
@@ -73,4 +73,66 @@ describes.realWin('UserActivationTracker', {}, env => {
     expect(tracker.isActive()).to.be.true;
     expect(tracker.getLastActivationTime()).to.equal(ACTIVATION_TIMEOUT + 2);
   });
+
+  describe('expandLongTask', () => {
+    it('should expand when active', () => {
+      const promise = Promise.resolve();
+
+      tracker.activated_({isTrusted: true});
+      tracker.expandLongTask(promise);
+      expect(tracker.isActive()).to.be.true;
+      expect(tracker.isInLongTask()).to.be.true;
+
+      clock.tick(ACTIVATION_TIMEOUT + 1);
+      expect(tracker.isActive()).to.be.true;
+
+      clock.tick(ACTIVATION_TIMEOUT + 1);
+      expect(tracker.isActive()).to.be.true;
+
+      return promise.then(() => {
+        // Skip microtask.
+        return Promise.resolve();
```

Comment:
Nit: Yield macrotask e.g. `poll(1)` may be less brittle.

Reply:
What's `poll(1)`? Do you mean `delay(1)`?

Reply:
I think I got it. Updated the code.